src/OrchardCore.Modules/OrchardCore.Users/TwoFactorAuthenticationStartup.cs

```diff
@@ -87,6 +115,25 @@ public override void ConfigureServices(IServiceCollection services)
         services.AddScoped<IDisplayDriver<ISite>, AuthenticatorAppLoginSettingsDisplayDriver>();
         services.AddScoped<IDisplayDriver<TwoFactorMethod>, TwoFactorMethodLoginAuthenticationAppDisplayDriver>();
     }
+
+    public override void Configure(IApplicationBuilder app, IEndpointRouteBuilder routes, IServiceProvider serviceProvider)
+    {
+        var _controllerName = typeof(AuthenticatorAppController).ControllerName();
```

Comment:
These variables, also below, shouldn't start with underscored.

Reply:
Yes. Maybe I was thinking it's global :)